packages/xo-web/src/xo-app/xoa/support/index.js

```diff
@@ -46,6 +47,18 @@ const Support = decorate([
           </Col>
         </Row>
       )}
+      <Row className='mb-1'>
+        <Col>
+          <ActionButton
+            btnStyle='primary'
+            disabled={COMMUNITY}
+            handler={reportOnSupportPanel}
+            icon='add'
```

Comment:
We can use [`fa-ticket`](https://fontawesome.com/v4.7.0/icon/ticket) since the support panel also uses it.